src/components/VolunteerDashbord/IndividualVolunteers/index.js

```diff
@@ -141,6 +141,15 @@ function VolunteerOverview(props) {
           <tbody>
             {classes && classes.length > 0 ? (
               slicedClasses.map((item) => {
+                let ratingCount = 0;
+                item.ratings.map((item) => {
+                  if (item.rating) ratingCount += parseInt(item.rating);
+
+                  return ratingCount;
+                });
+                item.avg_rating = Math.ceil(
+                  item.ratings.length && ratingCount / item.ratings.length
+                );
```

Comment:
I don't know what the data looks like here so this could be off base, but based on what you wrote, something like this should work:
```suggestion
                const ratings = item.ratings.filter((item) => item.rating);
                item.avg_rating = ratings.length && Math.ceil(
                    ratings.reduce((ratingSum, item) =>
                    ratingSum + parseInt(item.rating), 0)/ratings.length
                    );
```
Here, by using `filter`, we can extract the actual `ratings` and store it in an Array `ratings`.  This means we won't have to use the trick above and can just divide the sum by the `length` of this `ratings` Array as it represents the list of only valid ratings.  Here, we used `reduce` for computing the sum, a common use case.

Please check this before committing though because I don't know what form the data is taking.

Reply:
This is in fact correct suggestion. It also resolves the average_rating problem we discussed above. But this is for average_rating per class. We still need to resolve the average_rating per volunteer. This is the structure of ratings key
`       "ratings": [
          {
            "rating": "5"
          },
          {
            "rating": null
          },
          {
            "rating": "3"
          }
        ]`
@komalahire

Reply:
@jschanker Instead of `const avg_rating` we need to do `item.avg_rating`

Reply:
@pysaquib Yeah, I had used `avg_rating` for some quick console testing and had initially forgotten to change it back.  I updated my answer about an hour later, but I don't think notifications are sent for updates.  Anyway, I think the edited answer should be good.